WooCommerce/Classes/POS/Colors/Color+POSColorPalette.swift

```diff
@@ -0,0 +1,54 @@
+import SwiftUI
+
+extension Color {
```

Comment:
So many colors 😀 Fingers crossed it will be comfortable to work with 🤞

Reply:
31 new colors 😄 as long as the design uses it, it's been great not having to worry about dark mode.